### PR TITLE
Reddit changes

### PR DESCRIPTION
**Pull request by reverb5264**

### CODE REVIEW

**Review by AnonymousRandomPerson — CHANGES_REQUESTED** [review]
Resolve merge conflicts.

File: web/atlas.json [hunk 1]
```diff
@@ -1043,7 +1043,7 @@
...
-{"id": "twr7jt", "submitted_by": "GreatHeroJ", "name": "Zombie Face (Unturned)", "description": "The iconic face of a zombie from Unturned. This face doubles as the game's main logo.", "website": "https://store.steampowered.com/app/304930/Unturned/", "subreddit": "/r/unturned", "center": [670.5,583.5], "path": [[661.5, 574.5], [678.5, 574.5], [678.5, 591.5], [661.5, 591.5]]},
+{"id": "twr7jt", "submitted_by": "GreatHeroJ", "name": "Zombie Face (Unturned)", "description": "The iconic face of a zombie from Unturned. This face doubles as the game's main logo.", "website": "", "subreddit": "/r/unturned", "center": [670.5,583.5], "path": [[661.5, 574.5], [678.5, 574.5], [678.5, 591.5], [661.5, 591.5]]},
```
Comment (AnonymousRandomPerson) [inline web/atlas.json:1046]: This is unchanged other than removing the Steam link. Is this intended?

File: web/atlas.json [hunk 2]
```diff
@@ -2113,11 +2113,11 @@
...
-{"id": "twwf4i", "submitted_by": "Notx_Roh", "name": "Sportshot (Unturned)", "description": "In Unturned, the Sportshot is one of the game's most infamous weapons.", "website": "https://store.steampowered.com/app/304930/Unturned/", "subreddit": "/r/unturned", "center": [655.5, 589.5], "path": [[649.5, 591.5], [657.5, 591.5], [657.5, 590.5], [658.5, 590.5], [658.5, 589.5], [661.5, 589.5], [661.5, 587.5], [652.5, 587.5],[652.5, 588.5],[649.5, 588.5]]},
+{"id": "twwf4i", "submitted_by": "Notx_Roh", "name": "Sportshot (Unturned)", "description": "In Unturned, the Sportshot is one of the game's most infamous weapons.", "website": "", "subreddit": "", "center": [655.5, 589.5], "path": [[649.5, 591.5], [657.5, 591.5], [657.5, 590.5], [658.5, 590.5], [658.5, 589.5], [661.5, 589.5], [661.5, 587.5], [652.5, 587.5],[652.5, 588.5],[649.5, 588.5]]},
```
Comment (AnonymousRandomPerson) [inline web/atlas.json:2116]: This is unchanged other than removing the website and subreddit. Is this intended?

File: web/atlas.json [hunk 3]
```diff
@@ -3829,7 +3829,7 @@
...
-{"id": "tx5zl9", "submitted_by": "GreatHeroJ", "name": "Griefer Face (Unturned)", "description": "The 'griefer' face is a face that is usable by players in Unturned. It is Unturned's parody of the incredibly popular Trollface meme.", "website": "https://store.steampowered.com/app/304930/Unturned/", "subreddit": "/r/unturned", "center": [670.5, 566.5], "path": [[661.5, 557.5], [678.5, 557.5], [678.5, 574.5], [661.5, 574.5]]},
+{"id": "tx5zl9", "submitted_by": "GreatHeroJ", "name": "Griefer Face (Unturned)", "description": "The 'griefer' face is a face usable by players in Unturned. It is Unturned's parody of the incredibly popular Trollface meme.", "website": "", "subreddit": "", "center": [670.5, 566.5], "path": [[661.5, 557.5], [678.5, 557.5], [678.5, 574.5], [661.5, 574.5]]},
```
Comment (AnonymousRandomPerson) [inline web/atlas.json:3832]: Is this supposed to remove the website and subreddit?

Comment (reverb5264) [reply web/atlas.json:3832]: All three you mentioned were from this post, which just noticed they got back to me on https://www.reddit.com/r/placeAtlas2/comments/txt2um/entry_edits_unturned_zombie_face_griefer_face/

**Pull request closed without merging**